docs/blog/_posts/2021-02-17-scala3-rc1.md

```diff
@@ -104,18 +111,26 @@ Results in:
 
 As the warning suggests, now you should write `Foo.apply` instead of `Foo`. See [Issue #6190](https://github.com/lampepfl/dotty/issues/6190) and [PR #7207](https://github.com/lampepfl/dotty/pull/7207) for discussion.
 
-# Settling on `scaladoc` as the documentation tool
-We have settled on using the well-known `scaladoc` as a name for the documentation tool for Scala 3 (known previously as `scala3doc`).. The obsolete `dotty-doc` (or `scala3-doc`) is removed in RC1. We have also removed all the Kotlin dependencies (Dokka, etc.) from scaladoc. For details, see [PR #11349](https://github.com/lampepfl/dotty/pull/11349). To read more about `scaladoc`, see [documentation](https://dotty.epfl.ch/docs/usage/scaladoc/index.html)
+## Settling on `scaladoc` as the documentation tool
+
```

Comment:
What's the motivation behind using heading level 2 instead of 1 and leaving a newline?

Reply:
@anatoliykmetyuk I invite you to look at the [Google Style Guide](https://google.github.io/styleguide/docguide/style.html), for instance the section [Add spacing to headings](
https://google.github.io/styleguide/docguide/style.html#add-spacing-to-headings).
IMO the best choice would be to adher to an existing style guide or (at least) to agree on a minimal set of rules which work both with Scaladoc *and* with tools such as Pandoc (see PR #11257).
.